torch/csrc/jit/frontend/script_type_parser.cpp

```diff
@@ -190,6 +187,15 @@ TypePtr ScriptTypeParser::parseTypeFromExprImpl(const Expr& expr) const {
     }
     return subscriptToType(*value_name, subscript);
 
+  } else if (expr.kind() == TK_STRINGLITERAL) {
+    auto type_name = StringLiteral(expr).text();
```

Comment:
nit and optional:

Though `auto` will be deduced correctly, it is better to still include `const` and `&` for better readability.